DeviceAdapters/Utilities/DAGalvo.cpp

```diff
@@ -262,8 +283,14 @@ int DAGalvo::SetPolygonRepetitions(int repetitions)
 
 int DAGalvo::RunPolygons()
 {
-   return DEVICE_NOT_YET_IMPLEMENTED;
+   for (int i = 0; i < polygons_->size(); i++) {
+      DAPolygon* polygon = polygons_->at(i);
+      std::pair<double, double> xyPair = polygon->getVertex(0);
```

Comment:
Did you mean `getVertex(i)`?

Reply:
Never mind, I guess this is intentional(?)

Reply:
Yup, so far it only targets points, simplified by only using the first entry of the polygon.  Could be extended in the future to draw the complete polygon.

Sorry for merging so fast!  I could have avoided some problems by waiting.